common/djangoapps/adaptive_learning/config/tests/utils.py

```diff
@@ -0,0 +1,25 @@
+"""
+Provides helper functions for tests which need
+to configure flags related to Adaptive Learning.
+"""
+from contextlib import contextmanager
+
+from adaptive_learning.config.models import (
+        AdaptiveLearningEnabledFlag,
+        CourseAdaptiveLearningFlag
+)
+from request_cache.middleware import RequestCache
+
+
+@contextmanager
+def adaptive_learning_enabled_feature_flags(
+        global_flag,
+        course_id=None,
+        enabled_for_course=False
+):
```

Comment:
@tomaszgy This function is missing a docstring.

Reply:
👍